deps/openssl/config/README.md

```diff
@@ -0,0 +1,118 @@
+## Upgrading OpenSSL-1.1.0
+
+### Requirements
+- Linux enviroment (Only CentOS7.1 and Ubuntu16 are tested)
+- perl5
+- nasm (http://www.nasm.us/)  The version of 2.11 or higher is needed.
+- GNU as in binutils. The version of 2.26 or higher is needed.
+
+### 0. Check Requirements
+
+```
+$ perl -v
+
+This is perl 5, version 22, subversion 1 (v5.22.1) built for
+x86_64-linux-gnu-thread-multi
+(with 60 registered patches, see perl -V for more detail)
+
+$ as --version
+GNU assembler (GNU Binutils for Ubuntu) 2.26.1
+Copyright (C) 2015 Free Software Foundation, Inc.
+...
+$ nasm -v
+NASM version 2.11.08
+```
+### 1. Obtain and extract new OpenSS sources.
```

Comment:
`sources.` -> `sources` for consistency with other headings?

Reply:
Fixed.